jwala-persistence/src/test/java/com/cerner/jwala/persistence/jpa/service/app/impl/ApplicationCrudServiceImplTest.java

```diff
@@ -89,8 +89,10 @@ public MediaDao getMediaDao(){
 
         @Bean
         public GroupPersistenceService getGroupPersistenceService() {
-            return new JpaGroupPersistenceServiceImpl(getGroupCrudService(),
-                    getGroupJvmRelationshipService());
+            return new JpaGroupPersistenceServiceImpl(
```

Comment:
Minor: I know you didn't format it this way since you just added a parameter but it would look nicer if it were formatted like this:

```java
return new JpaGroupPersistenceServiceImpl(getGroupCrudService(), getGroupJvmRelationshipService(), 
                                          getApplicationCrudService());
```